src/twisted/words/protocols/irc.py

```diff
@@ -2650,9 +2661,18 @@ def dataReceived(self, data):
         basic.LineReceiver.dataReceived(self, data)
 
     def lineReceived(self, line):
-        if bytes != str and isinstance(line, bytes):
-            # decode bytes from transport to unicode
-            line = line.decode("utf-8")
+        if isinstance(line, bytes):
```

Comment:
It will _always_ be `bytes` &mdash; please remove this `isinstance` condition.

Reply:
Removing this check broke some other tests that called `lineReceived` directly, but I fixed those.

Reply:
Perfect, thank you!